src/woo_publications/publications/api/serializers.py

```diff
@@ -364,6 +368,10 @@ def create(self, validated_data):
         )
         document = super().create(validated_data)
 
+        if document.publicatiestatus == PublicationStatusOptions.published:
+            document.gepubliceerd_op = document.laatst_gewijzigd_datum
+            document.save()
```

Comment:
Nice touch to set this to the exact same timestamp, but can't this be done in a single save?